core/common/src/ChangedElements.ts

```diff
@@ -2,35 +2,79 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
+/**
+ * @packageDocumentation
+ * @module Entities
+ */
 import { Id64String } from "@bentley/bentleyjs-core";
 import { AxisAlignedBox3dProps } from "./geometry/Placement";
 
 // cspell:ignore bboxes
 
-/** @packageDocumentation
- * @module Entities
+/** Bitflags describing which aspects of an [Element]($backend) changed as part of a [[ChangedElements]].
+ * @public
  */
-
-/** @internal */
 export enum TypeOfChange {
+  /** A property in the element changed */
   Property = 0b1,
+  /** The geometry stream of a [GeometricElement]($backend) changed */
   Geometry = 0b10,
+  /** The [[Placement]] of a [GeometricElement]($backend) changed */
   Placement = 0b100,
+  /** Indirect change occurred to this element by a related instance */
   Indirect = 0b1000,
+  /** Hidden properties of the element changed */
   Hidden = 0b10000,
 }
 
-/** @internal */
+/** Changed elements found in a changeset or between a range of changesets
+ * All arrays in this object will have the same number of items
+ * Each index for those arrays refer to the same element, e.g. to get the class Id of
+ * the element given by element Id changedElements.elements[index], you can use changedElements.classIds[index]
+ * @public
+ */
 export interface ChangedElements {
+  /** Ids of elements that changed */
   elements: Id64String[];
+  /** ECClass Ids of elements that changed */
   classIds: Id64String[];
+  /** Operation that occurred on the element. Whether the element was inserted, updated or deleted.
+   * See [DbOpcode]($bentleyjs-core)
+   */
   opcodes: number[];
+  /** Type of change bitflags, see [[TypeOfChange]] */
   type: number[];
```

Comment:
If this is literally of type `TypeOfChange` please change it from `number`.

Reply:
Pushed